src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1166,6 +1166,31 @@ public function pipesCodeProvider()
     }
 
     /**
+     * @dataProvider provideVariousIncrementals
+     */
+    public function testIncrementalOutputDoesNotRequiresAnotherCall($stream, $method) {
+        $process = new Process(self::$phpBin." -r '\$n = 0; while (\$n < 3) { file_put_contents(\"$stream\", \$n, 1); \$n++; usleep(1000); }'", null, null, null, null);
```

Comment:
I think the issue on Windows might be related to the double quotes in the code. I suggest using single quotes and escapeshellargs as done for other tests